js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -58,70 +61,102 @@ export const LaunchAssetExecutionButton: React.FC<{
       disabledReason || 'Assets must be in the same job to be materialized together.';
   }
 
+  const assetsWithConfig = assets.filter((a) => configSchemaForAssetNode(a));
+  const anyAssetsHaveConfig = assetsWithConfig.length > 0;
+
+  if (anyAssetsHaveConfig && partitionDefinition) {
+    disabledReason =
+      disabledReason || 'Cannot materialize assets using both asset config and partitions';
+  }
+
   title = title || 'Refresh';
-  if (partitionDefinition) {
+
+  let tooltipChildren: React.ReactNode;
+  if (anyAssetsHaveConfig) {
+    const assetOpNames = assets.map(({opNames}) => (opNames as string[])[0]);
+    const sessionPresets = {
+      solidSelection: assetOpNames,
+      solidSelectionQuery: assetOpNames.map((name) => `"${name}"`).join(' '),
```

Comment:
⚠️  I think this has changed since your PR was started actually - you should probably merge in the switch to `assetSelection` instead of `solidSelection` (from line 117 of the old version). Hopefully it makes my comment up above unnecessary!

Reply:
Discussed this offline and we'll just fix it later, only impacts this new config case!